masterlist.yaml

```
@@ -12708,6 +12708,28 @@ plugins:
         util: '[TES4Edit v4.0.3g](https://www.nexusmods.com/oblivion/mods/11536)'
         udr: 15
 
+  - name: 'CloudTopManor.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/46148/'
+        name: 'Beneath Cloud Top Manor'
+    req:
+      - *OBSE
+      - 'DLCShiveringIsles.esp'
+    inc:
```

The merged ESP includes these, so it can be added too:

```suggestion
    inc:
      - name: 'Unique Landscapes.esp'
        condition: 'version("Unique Landscapes.esp", "2.0.0", >=)'
```